EXERCISE-2.md

```diff
@@ -230,6 +230,27 @@ Starting onos    ... done
 You will need to go back to Exercise 1 and install forwarding rules to
 re-establish pings between `h1a` and `h1b` for later parts of this exercise.
 
+If you could not complete Exercise 1, you can use the following P4Runtime-sh
+commands to enable connectivity:
+
+<details><summary>Solution Exercise 1</summary>
```

Comment:
We don't use html tags anywhere else in the instructions (besides for images). The reason is that we want instructions to be consumable from an IDE or terminal, i.e. not rendered on GitHub. It's easier to read without HTML tags.

Reply:
Got it. Will revert to plain markdown.